api/src/test/java/uk/gov/dhsc/htbhf/claimant/testsupport/ClaimantTestDataFactory.java

```diff
@@ -17,6 +17,7 @@
     private static final String VALID_FIRST_NAME = "James";
     private static final String VALID_LAST_NAME = "Smith";
     private static final LocalDate VALID_DOB = LocalDate.parse("1987-12-30");
+    private static final String VALID_PHONE_NUMBER = "+447700900128";
```

Comment:
I notice that above and here we are talking about phone numbers starting with +44, is this something we are asking the applicants to do, I'd expect them to simply start with 0, or are we converting it to have the international code? Seems a bit odd to me.

Reply:
In the kick off we decided we'd store the number in +44 format for consistency. We chose +44 over 0 because one of the card providers said they required numbers in that format.